src/Rating.js

```diff
@@ -1,8 +1,228 @@
 'use strict';
 
 import RatingBase from './Rating.soy';
+import { core } from 'metal';
 
 class Rating extends RatingBase {
```

Comment:
Another thing about the new version is that we've changed how soy components are created so that they don't need to extend from the soy file (that could be confusing sometimes if you needed to extend from somewhere else).

So when you update the default export the [soy file will be the template functions instead](https://github.com/metal/metal-progressbar/blob/master/src/ProgressBar.js#L6). Your component should [extend directly from `Component`](https://github.com/metal/metal-progressbar/blob/master/src/ProgressBar.js#L11) now and you should [make a call to `Soy.register`](https://github.com/metal/metal-progressbar/blob/master/src/ProgressBar.js#L105) to indicate that you'll be using soy templates.

Let me know if you have any problems updating :)